circuits/bigint.circom

```diff
@@ -172,6 +172,31 @@ template BigAdd(n, k) {
     out[k] <== unit[k - 2].carry;
 }
 
+template BigAddNoCarry(n, k) {
```

Comment:
the only difference from `BigAdd` is that its `out[k]` instead of `out[k+1]` removing the constraint with the last carry bit.